avro_schema/compiler.lua

```diff
@@ -162,8 +162,12 @@ local schema2ilfunc = {
 local function append_put_value(il, flat, code, schema, value)
     local n = #code
     code[n + 1] = il.checkobuf(1)
-    local ilfunc = schema2ilfunc[schema]
-    if ilfunc then
+    local ilfunc = schema2ilfunc[schema.nullable and schema.type or schema]
```

Comment:
commit 'Fix nullable default values': 'non-nullabe' in the commit message should be 'non-nullable'.

Reply:
fixed